### ISSUE TITLE
[BUG] Online Grayskull tool has expired certificate

### ISSUE THREAD

**Issue by moqmar** [conda/grayskull#462]
**Describe the bug**
I get the error "Failed to Fetch" when trying to use the online grayskull tool.

Certificate on marcelotrevisani.info:8000 is expired since Jan 27: https://decoder.link/sslchecker/marcelotrevisani.info/8000 @marcelotrevisani 

**To Reproduce**
1. Go to 'https://www.marcelotrevisani.com/grayskull'
2. Enter the project name "numpy"
3. Click "Generate Recipe"

**Expected behavior**
No error message.

**Outputs**
```
GET https://marcelotrevisani.info:8000/recipe/?name=tftpy net::ERR_CERT_DATE_INVALID
```

**Environment:** N/A

**Additional context:** N/A

**Comment by marcelotrevisani** [comment]
yes, I still need to find some time to fix this, thanks for reporting it
